src/main/resources/static/oas/basketball.yaml

```diff
@@ -1,34 +1,1358 @@
+      parameters:
+        - name: season_id
+          in: query
+          required: false
+          schema:
+            type: integer
+          description: Filter by season ID
+      responses:
+        '200':
+          description: Average points stats retrieved successfully
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/TeamStats'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /stats/teams/win-rate:
+    get:
+      tags: [ Stats ]
+      summary: Get teams by win rate
+      parameters:
+        - name: season_id
+          in: query
+          required: false
+          schema:
+            type: integer
+          description: Filter by season ID
+      responses:
+        '200':
+          description: Win rate stats retrieved successfully
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/TeamStats'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /stats/matches/highest-scoring:
+    get:
+      tags: [ Stats ]
+      summary: Get highest scoring matches
+      parameters:
+        - name: season_id
+          in: query
+          required: false
+          schema:
+            type: integer
+          description: Filter by season ID
+      responses:
+        '200':
+          description: Highest scoring matches retrieved successfully
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Match'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+components:
+  schemas:
+    User:
+      type: object
+      properties:
+        id:
+          type: integer
+        login:
+          type: string
+        password:
+          type: string
+        first_name:
+          type: string
+        last_name:
+          type: string
+        role:
+          type: string
+          enum: [ admin, user ]
+      required: [ id, login, password, role ]
+
+    Player:
+      type: object
+      properties:
+        id:
+          type: integer
+        user_id:
+          type: integer
+        team_id:
+          type: integer
+        first_name:
+          type: string
+        last_name:
+          type: string
+        position:
+          type: string
+        age:
+          type: integer
+        height_cm:
```

Comment:
let's not make `cm` part of our code, just document it in description and make it simple - `heigh`